src/Swisstopo.js

```diff
@@ -102,8 +114,8 @@ class SwisstopoSource extends olSourceWMTS {
     console.assert(!!extension);
 
     super({
-      attributions: '&copy; <a href="http://www.swisstopo.admin.ch">swisstopo</a>',
-      url: swisstopoCreateUrl(projection, extension),
+      attributions: options.attributions || DEFAULT_ATTRIBUTIONS,
+      url: `${options.baseUrl || DEFAULT_BASE_URL}${createUrl(projection, extension)}`,
```

Comment:
Instead of doing the concatenation here, I would base the base URL to the createURL function and return the concatenated URL. I think it would be more clear.